package.json

```diff
@@ -88,10 +91,12 @@
     "passport-github": "^1.1.0",
     "puppeteer": "^1.8.0",
     "react": "^16.5.0",
+    "react-codemirror2": "^5.1.0",
     "react-dom": "^16.5.0",
     "redis": "^2.8.0",
     "styled-components": "^3.4.5",
     "styled-normalize": "^8.0.0",
+    "tohash": "^1.0.2",
```

Comment:
Why `tohash ` is needed?

Reply:
I used for convenience.

Reply:
Why don't use rebase?

Reply:
You mean git rebase?

Reply:
this merge request have many merge commit. i think one merge commit  to one merge request. so @thomasJang say why don't use rebase.  when you send merge request, i recommend "git pull --rebase origin master" in your branch.  this command your commit list merge and rebase easy before you send merge request.